packages/trader/src/Stores/Helpers/client-notifications.js

```diff
@@ -203,6 +215,60 @@ export const clientNotifications = {
         ),
         type: 'info',
     },
+    poa_expired: {
+        key    : 'poa_expired',
+        header : localize('Document expired'),
+        message: (
+            <Localize
+                i18n_default_text='Your documents for proof of address is expired. Please <0>submit</0> again.'
+                components={[
+                    <BinaryLink
+                        key={0}
+                        className='link link--white'
+                        to='account/proof-of-address'
+                    />,
+                ]}
+            />
+        ),
+        type: 'danger',
+    },
+    poa_rejected: {
+        key    : 'poa_rejected',
+        header : localize('We could not verify your proof of address'),
+        message: (
+            <Localize
+                i18n_default_text='We have disabled trading, deposits and withdrawals for this account.'
+            />
+        ),
+        type: 'danger',
+    },
+    poi_expired: {
+        key    : 'poi_expired',
+        header : localize('Proof of identity expired'),
+        message: (
+            <Localize
+                i18n_default_text='Your proof of identity document has expired. Please <0>submit</0> a new one.'
+                components={[
+                    <BinaryLink
+                        key={0}
+                        className='link link--white'
+                        to='account/proof-of-identity'
+                    />,
+                ]}
+            />
+        ),
+        type: 'danger',
+    },
+    poi_rejected: {
+        key    : 'poi_rejected',
+        header : localize('We could not verify your proof of identity'),
+        message: (
+            <Localize
+                i18n_default_text='We have disabled trading, deposits and withdrawals for this account.'
+            />
+        ),
+        type: 'danger',
+    },
```

Comment:
will there any message notification if poi or poa is needed, but they havent submit document?

Reply:
@mustofa-binary yes, we'll show a notification if authentication is required. you can refer to the line here: https://github.com/binary-com/deriv-app/blob/dev/packages/trader/src/Stores/Helpers/client-notifications.js#L41